doc/api/cli.md

```diff
@@ -568,10 +568,32 @@ appended to if it does. If an error occurs while attempting to write the
 warning to the file, the warning will be written to stderr instead. This is
 equivalent to using the `--redirect-warnings=file` command-line flag.
 
+### `UV_THREADPOOL_SIZE=size`
+
+Set the number of threads used in libuv's threadpool to `size` threads.
+
+Asynchronous system APIs are used by Node.js whenever possible, but where they
+do not exist, libuv's threadpool is used to create asynchronous node APIs based
+on synchronous system APIs. Node.js APIs that use the threadpool are:
+
+- `fs.*()`
+- `dns.lookup()`
+- `crypto.randomBytes()`
```

Comment:
An asterisk or something should be added here to note this is only when a callback is supplied, which activates async mode.